Venafi/vcert
# Playbook cucumber tests

This PR includes:
-  changes to grab the trust bundle file from aruba work directory
- test coverage for following scenarios:
   - Run playbook for TPP with extended configuration with PEM, PKCS12 and PEM installations
   - Run playbook with default configuration, perform simple PEM installation and validates private key
   - Run playbook with default configuration, perform simple PEM installation and validates encrypted private key
   - Run playbook twice with default configuration, perform simple PEM installation. Should prevent second issue
   - Run playbook twice with default configuration, perform simple PEM installation. Should issue twice
   - Run playbook twice with default configuration and --force-renew flag, perform simple PEM installation. Should issue twice
   - Run playbook with default configuration, perform two tasks, each one doing PEM and PKCS12 installations respectively

closes VC-24598
